package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@onfido/api",
+  "version": "0.1.0",
+  "description": "Node.js library for the Onfido API",
+  "keywords": ["onfido", "identity", "verification", "api"],
+  "homepage": "https://github.com/onfido/onfido-node#readme",
+  "license": "MIT",
+  "author": "Onfido <api@onfido.com> (https://documentation.onfido.com)",
+  "repository": "github:onfido/onfido-node",
+  "main": "dist/index.js",
+  "module": "dist/index.es.js",
+  "types": "dist/index.d.ts",
+  "sideEffects": false,
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "rollup -c rollup.config.ts",
+    "test": "jest",
+    "test:debug": "node --inspect-brk node_modules/.bin/jest --runInBand",
```

Comment:
[Minor] I don't believe you need to set the path to Jest executable - just `jest` should work?

Reply:
Think you do for debugging, doesn't work without it: https://jestjs.io/docs/en/troubleshooting#tests-are-failing-and-you-dont-know-why